<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_two_dir</class>
 <widget class="QDialog" name="choose_two_dir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/image.qrc">
    <normaloff>:/icon/camera.png</normaloff>:/icon/camera.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>200</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>选择左侧相机的图片的文件夹</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="left_src">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_left">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="right_src">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>200</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>选择右侧相机的图片的文件夹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_right">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>155</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>角点间距：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="corner_size">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>155</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yes">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>190</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res/image.qrc"/>
 </resources>
 <connections/>
</ui>
